### ISSUE TITLE
java.lang.outofmemoryerror android

### ISSUE THREAD

**Issue by dev-aminmo** [btastic/flutter_native_image#91]
when trying to compress high-resolution images this error occurs in release-mode only

**Comment by ModernaVaccine** [comment]
Yeah this library has the same issue as the standard flutter image library. It runs out of memory, upon loading up the original image on Android. Would there be a way to add in bitmap sample sizes like shown in this article? https://developer.android.com/topic/performance/graphics
